examples/README.md

```diff
@@ -95,19 +94,25 @@ Loads data directly from an URL.
 
 ## [From JSON spec](https://github.com/procyon-rs/vega_lite_3.rs/blob/master/examples/from_json_spec.rs)
 
-```
+```rust
 cargo run --example from_json_spec
 ```
 
 Load the chart directly from the JSON spec.
 
 <img src="https://raw.githubusercontent.com/procyon-rs/vega_lite_3.rs/master/examples/res/screens/diverging_stacked_bar_chart.png">
 
-# Miscellaneous
+```rust
```

Comment:
this sample is missing a title, a link to the source file and an image

Reply:
As this was the same type of graph as the ndarray one i didn't feel the need to do it but I agree there should also be a title, link and image.